src/components/FAQ.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react'
+import { Link } from 'react-scroll'
+import './FAQ.css'
+
+function Accordian({faq, index, toggleFAQ}) {
+    return (
+        <div className={"faq " + (faq.open ? "open" : "")} key={index} onClick={() => toggleFAQ(index)}>
+            <div className="faq-question">
+                {faq.question}
+                <i className={"faq-icon fas fa-xs " + (faq.open ? "fa-minus" : "fa-plus")}></i>
+            </div>
+            <div className="faq-answer">
+                {faq.answer1}{faq.link}{faq.answer2}
+            </div>
+        </div>
+    )
+}
+
+function FAQ () {
+    const [faqs, setFaqs] = useState([
+        {
+            question: "What is SBUHacks?",
+            answer1: "SBUHacks is a hackathon where students are challenged to further what they know by doing projects on what they are passionate about. Come through to have fun with fellow students like you put together something unforgettable.",
+            link: null,
+            answer2: "",
+            open: false
```

Comment:
Rather than this check state of whats open using an array. Save an array of booleans to check what questions are open and closed. That way by simply checking array[questionNumber] you can see quickly whats open and whats closed, Save all this text data outside of state.